gn3/computations/rust_correlation.py

```diff
@@ -0,0 +1,91 @@
+"""module contains code integration  correlation implemented in rust here
+
+https://github.com/Alexanderlacuna/correlation_rust
+
+"""
+
+import subprocess
+import json
+import os
+
+
+from gn3.computations.qtlreaper import create_output_directory
+from gn3.random import random_string
+from gn3.settings import CORRELATION_COMMAND
+from gn3.settings import TMPDIR
+
+
+def generate_input_files(dataset: list[str], output_dir: str = TMPDIR):
```

Comment:
*suggestion: nitpick:* Let's try to be consistent with our naming. I see we have 2 different patterns: "generate_*" vs "create_*". We should choose one and stick to it.